### PR TITLE
[.NET 5] Added nuspec and nuget.config files

### PR DESCRIPTION
**Pull request by mdey1**
**Overview**: The purpose of this PR is to create a nuspec and nuget.config file for the PipeMethodCalls library, so that we can eventually create and publish its nuget package. 

**JIRA**: https://coveord.atlassian.net/browse/CT-7597

### CODE REVIEW

**Comment by comagnan** [comment]
Hello!

File: PipeMethodCalls.nuspec [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<package >
+  <metadata>
+    <id>PipeMethodCalls</id>
+    <version>$version$</version>
+    <authors>Coveo Connectors Team</authors>
+    <owners>The PipeMethodCalls library.</owners>
```
Comment (comagnan) [inline PipeMethodCalls.nuspec:7]: That'd be RandomEngy, basic myself off https://www.nuget.org/packages/PipeMethodCalls/

File: PipeMethodCalls.nuspec [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>Contains the PipeMethodCalls library.</description>
```
Comment (comagnan) [inline PipeMethodCalls.nuspec:9]: Could reuse the official description ( https://www.nuget.org/packages/PipeMethodCalls/ ) : `Lightweight library for method calls over named pipes for IPC in .NET Core. Supports two-way communication with callbacks.`

**Review by comagnan — APPROVED** [review]

File: PipeMethodCalls.nuspec [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    <copyright>Copyright (c), The PipeMethodCalls library authors.</copyright>
+    <dependencies>
+      <group targetFramework="netstandard2.1" />
```
Comment (ggingras) [inline PipeMethodCalls.nuspec:12]: We must target netstandard2.0, otherwise it will not be compatible with .NET 4.8

**Review by ggingras — APPROVED** [review]

**Pull request merged**